src/collective/cover/utils.py

```diff
@@ -17,3 +18,29 @@ def assign_tile_ids(layout, override=True):
             children = elem.get('children')
             if children:
                 assign_tile_ids(children, override)
+
+
+def uuidToObject(uuid):
+    """Return a content object given an UUID.
+
+    :param uuid: UUID of the object
+    :type uuid: str
+    :returns: the content object or None, if the uuid can't be found.
+    """
+    # Use local uuidToCatalogBrain without the inactive content filter
+    brain = uuidToCatalogBrain(uuid)
+    return brain.getObject() if brain else None
+
+
+def uuidToCatalogBrain(uuid):
```

Comment:
I intentionally kept the names the same at first because I copied them as a whole from plone.app.uuid . But I don't care that much about the naming. 

A suggestion on the plone-developers mailinglist was to add a keyword agrument to this function in plone.app.uuid so that you can disable the filter, but that doesn't help c.cover in the short term, it would be strange to depend on a higher p.a.uuid if we support 4.2/4.3/5 with collective.cover. But in the long term we could then use the plone.app.uuid again.